### PR TITLE
cmd/faucet: fix readme typos

### PR DESCRIPTION
**Pull request by omahs**
Fix: typos

### CODE REVIEW

File: cmd/faucet/README.md [hunk 1]
```diff
@@ -50,4 +50,4 @@ Sybil protection via Facebook uses the website to directly download post data th
 
 ## Miscellaneous
 
-Beside the above - mostly essential - CLI flags, there are a number that can be used to fine tune the `faucet`'s operation. Please see `faucet --help` for a full list.
\ No newline at end of file
+Besides the above - mostly essential - CLI flags, there is a number that can be used to fine-tune the `faucet`'s operation. Please see `faucet --help` for a full list.
```
Comment (holiman) [inline cmd/faucet/README.md:53]: "Beside [the flags] above" vs "Besides [the flags] above" -- former seems more correct to me
"there are a number [of flags] that can be used" vs
"there is a number [of flags] that can be used". Again, former seems more correct

Comment (omahs) [reply cmd/faucet/README.md]: Ok! Just edited that part

**Review by holiman — APPROVED** [review]
LGTM

**Comment by holiman** [comment]
thanks!

**Pull request merged**